src/stars.css

```diff
@@ -0,0 +1,75 @@
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-size: 13px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+/* #region star--0 */
+.stars--0 .stars__star:nth-child(-n + 5) {
+  background-image: url(/src/images/star.svg);
+}
+
+/* #endregion */
+
+/* #region stars--1 */
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-last-child(-n + 4) {
+  background-image: url(/src/images/star.svg);
+}
+
+/* #endregion */
+
+/* #region stars--2 */
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-last-child(-n + 3) {
+  background-image: url(/src/images/star.svg);
+}
+
+/* #endregion */
+
+/* #region stars--3 */
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-last-child(-n + 2) {
+  background-image: url(/src/images/star.svg);
+}
+
+/* #endregion */
+
+/* #region stars--4 */
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-last-child(-n + 1) {
+  background-image: url(/src/images/star.svg);
+}
+
+/* #endregion */
+
+/* #region stars--5 */
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+/* #endregion */
```

Comment:
you are in al that selectors using same css rule, just combine this selectors to avoid code duplications, also remove regions after that